docs/endpoints/openrtb2/amp.md

```diff
@@ -0,0 +1,41 @@
+# Prebid Server AMP Endpoint
+
+This document describes the behavior of the Prebid Server amp endpoint, including:
+
+- Request/response formats
+
+## AMP RTC
+Legacy support for prebid in AMP is being retired. RTC (Real Time Config) in AMP provides a new route
+to inject prebid demand into an AMP page. RTC can only send a `GET` request, and only recieve ad 
+targeting key/value pairs, so the prebid AMP protocol is a slimmed down version of the OpenRTB2
+protocol. This document will only describe the changes from OpenRTB2 support, please refer to that
+documentation for anything not specified here.
+
+## `GET /openrtb2/amp?tag_id=<ID>`
```

Comment:
How about account ID? We're quite careful with regular Prebid Server requests to track and validate incoming requests by account.

Reply:
We haven't really incorporated the concept of account into the OpenRTB code. I am not sure what adding an account ID will give us beyond allowing tag_id's to be non-unique across different accounts.